src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,22 @@
 package core.mate.academy.model;
 
 /**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
+ * Add some custom fields that could be only in Bulldozer Do not remove no-args constructor
```

Comment:
The comment mentions adding custom fields specific to `Bulldozer`. Ensure that `bladeWidth` is the only custom field needed. If there are other unique attributes for a bulldozer, consider adding them as well.